samples/snippets/speech_model_adaptation_beta.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START speech_transcribe_with_model_adaptation]
+
+from google.cloud import speech_v1p1beta1 as speech
+
+
+def transcribe_with_model_adaptation(project_id, location, storage_uri):
+
+    """
+    Create`PhraseSet` and `CustomClasses` to create custom lists of similar
+    items that are likely to occur in your input data.
+    """
+
+    # Create the adaptation client
+    adaptation_client = speech.AdaptationClient()
+
+    # The parent resource where the custom class and phrase set will be created.
+    parent = f"projects/{project_id}/locations/{location}"
+
+    # Create the custom class
+    custom_class_response = adaptation_client.create_custom_class(
+        {
+            "parent": parent,
+            "custom_class_id": "seattlerestaurants",
```

Comment:
Please make the ids for the class and phrase sets arguments to the function. That way the test can pass unique names and the tests will not clobber each other.